src/electionguard_gui/web/components/admin/view-key-ceremony-component.js

```diff
@@ -1,6 +1,13 @@
+import KeyCeremonyDetails from "../shared/key-ceremony-details-component.js";
+
 export default {
   props: {
     keyCeremonyId: String,
   },
-  template: /*html*/ `<h1>Key Ceremony: {{keyCeremonyId}}</h1>`,
+  components: {
+    KeyCeremonyDetails,
+  },
+  template: /*html*/ `
+    <key-ceremony-details :keyCeremonyId="keyCeremonyId"></key-ceremony-details>
```

Comment:
There are two separate key ceremony pages, one for the admin and one for the guardian.  They both display the same component.  This felt cleaner than having a shared page because it gives the opportunity to customize the page if needed.